index.md

```diff
@@ -17,11 +17,9 @@ order: 1
           <br>
 
           <hr class="mid-sepr">
-          <div class="" href="#one!">
-            <div class="quotation">
-              Viper Development helped us save at least 2 million euro revenue per year.
-            </div>
-            <div class="small-heading">Jozef Képesi, CTO at Skypicker</div>
+          <div class="small-heading">
```

Comment:
small-heading class is the style for subtitle and quotes, decreasing font variations